internal/command/kubernetes.go

```diff
@@ -0,0 +1,384 @@
+package command
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/fyne-io/fyne-cross/internal/cloud"
+	"github.com/fyne-io/fyne-cross/internal/log"
+	"github.com/fyne-io/fyne-cross/internal/volume"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I don't think the v1 is needed here is it, adding api version just clutters the code that uses it